src/common/SteelSheets.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+#include "marlin_client.h"
+#include "eeprom.h"
+
+class SteelSheets {
+public:
+    static constexpr float Z_OFFSET_MIN = -2.0F;
+    static constexpr float Z_OFFSET_MAX = 0.0F;
+
+    ///////////////////////////////////////////////////////////////////////////////
+    /// @brief Sets offset for sheet
+    ///
+    /// @param[in] index Index of the sheet profile
+    /// @param[in] offset of the sheet
+    /// @return True when successful, false if not
+    static bool SetSheetOffset(uint32_t index, float offset);
```

Comment:
`static bool SetSheetOffset(uint32_t index, float offset)` is not used outside, so it can be made private.